concreep-refilled/control.lua

```diff
@@ -171,15 +177,15 @@ function _reinit_reentrant_level1_body()
 					surface and
 					surface.valid
 				) then
-					_reinit_reentrant_level2_start(surface)
+					_reinit_reentrant_level2_start(surface) 
 				else
 					-- degenerate case, this is what increments the outer index
 					_reinit_reentrant_level2_end()
 				end
 			end
 			-- if we got here:
-			--  - we either exhausted the list of surfaces (so return true) or 
-			--  - we reached SURFACES_PER_REINIT_PASS for this call (so return false)
+			--  - we either exhausted the list of settings.global["allowed surfaces"].value (so return true) or 
+			--  - we reached DEFAULT_SURFACES_PER_REINIT_PASS for this call (so return false)

```

Comment:
should reference the correct settings.global here

Reply:
done